### DISCUSSION TITLE
Problems with docker

### DISCUSSION THREAD

**Discussion by xxtemps** [ansys/pyansys-geometry#1150] [Q&A] [answered]
![image](https://github.com/ansys/pyansys-geometry/assets/12379914/228a7c93-56a7-4884-83de-c6184aed7476)
What is the problem, please?

**Comment by RobPasMue** [comment, marked as answer]
Hi @xxtemps 

As it is explained in the documentation - you are tying to download the Geometry image from GitHub's container registry. However, this option is only available for Ansys employees. See the note in this section: https://geometry.docs.pyansys.com/version/stable/getting_started/docker/windows_container.html#github-container-registry

If you are not an Ansys employee and you want to use the Docker option to launch the Geometry service, you can build your own Geometry image based on your local Ansys installation. See https://geometry.docs.pyansys.com/version/stable/getting_started/docker/windows_container.html#build-the-geometry-service-windows-container for more information on how to build your own Docker image.

More specifically, you will have to follow the instructions in the [Build from available Ansys installation](https://geometry.docs.pyansys.com/version/stable/getting_started/docker/windows_container.html#build-from-available-ansys-installation) section.
